synchronization/src/main/java/de/cyface/synchronization/WiFiSurveyor.java

```diff
@@ -336,7 +335,7 @@ public void makeAccountSyncable(@NonNull final Account account, boolean enabled)
         ContentResolver.setSyncAutomatically(account, authority, false);
         setSyncEnabled(account, enabled);
 
-            // Do not use validateAccountFlags in production code as periodicSync flags are set async
+        // Do not use validateAccountFlags in production code as periodicSync flags are set async
```

Comment:
Comment has no context. What is validateAccountFlags?

Reply:
That's a method I moved into test code. Had this validation in here before when I still thought that periodicSyncFlags are set synchroniously. I'll pushed a commit which clarifies this comment.